### DISCUSSION TITLE
Hey Can we Improve the CLI scripts with typer

### DISCUSSION THREAD

**Discussion by Ether-Sagittarius** [FOSS-Community/progress-tracker#41] [Q&A] [answered]
Hey @Mr-Sunglasses the Current CLI uses argparse but we can use [typer](https://typer.tiangolo.com/)  to make it better and easy to use.

**Comment by Mr-Sunglasses** [comment, marked as answer]
Thanks @Ether-Sagittarius  for the Suggestion, I'll be Migrating to [type](https://typer.tiangolo.com/) as soon as possible.
